README.md

```diff
@@ -26,6 +26,24 @@ yarn
 yarn start
 ```
 
+However, it is possible to build the website locally using okteto itself without any need to install required dependencies and specific version of build tool.
+
+### 💻 Develop locally (using Okteto)
```

Comment:
This is great! Thanks for your contribution!

Reply:
Pleasure is mine:)